STYLE_GUIDE.md

```diff
@@ -119,7 +119,9 @@ color: rgba(black, 0.5);
 
 ### Theme colours
 
-Kedro-Viz has both light and dark themes, and every component must be styled appropriately for each theme.
+Kedro-Viz has both light and dark themes, and every component must be styled appropriately for each theme. Below is the style guide for kedro colors 
+
+![Kedro-Viz Color Styleguide](./.github/img/kedro_colours.png)
```

Comment:
I would reword this as  `Kedro-Viz has both light and dark themes, and every component must be styled appropriately for each theme. Please refer to ![Kedro-Viz Color Styleguide](./.github/img/kedro_colours.png) for the set of colours for each theme.`

Reply:
Hi, Susanna -- just for this one I am just saying _Please refer to below colour guide for the set of colors..._ 

Because it's an image below the text and not a link.